### PR TITLE
Removed Symfony extension, updated master branch alias

### LINKED ISSUE #49: Move Symfony2 extensions into datasource-bundle

### PR DESCRIPTION
**Pull request by szymach**
Resolves #49

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

**Comment by chives** [comment]
We should also remove all useless symfony components composer.json

File: CHANGELOG-2.0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changelog for version 2.0
+
+This is a list of changes done in version 2.0.
+
+## Deleted Symfony extension
+
+Since it was moved to [datasource-bundle](https://github.com/fsi-open/datasource-bundle),
+it has been removed from this bundle.
```
Comment (chives) [inline CHANGELOG-2.0.md]: `this bundle` => `the component`

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@rn0

**Pull request merged**